### ISSUE TITLE
Hotbar will stay revealed when holding food related items

### ISSUE THREAD

**Issue by GoodOleJar** [Crendgrim/AutoHUD#119]
Hotbar will not fadeout/move and will stay revealed if you're holding food related items
![2024-06-21_20 14 52](https://github.com/Crendgrim/AutoHUD/assets/107520126/6380567c-58e5-4475-8f3e-48eb06fdd6d8)

**Comment by Crendgrim** [comment]
This behaviour is on purpose: it shows your hunger when you have a food item and the food bar is not full.
I thought there was an option to disable that, though, and that does not appear to be the case.
Will work on this together with #118

**Comment by Jouto25** [comment]
> I thought there was an option to disable that, though, and that does not appear to be the case. Will work on this together with #118

If this option was added in the config, which option is it on the config menu? Because I checked the config menu, and the food bar is still showing when food items are selected.

![image](https://github.com/user-attachments/assets/6eca31c5-e173-4d23-8530-ffc46c933912)

![image](https://github.com/user-attachments/assets/62349e39-aad6-47a0-a344-f3999e168095)

This is on Fabric 0.6.9, MC 1.21.1, and AutoHUD 7.2.7.
